totp.go

```diff
@@ -0,0 +1,14 @@
+package clerk
+
+// TOTP describes a TOTP (Time-based One-Time Password) for a user.
```

Comment:
Super nit, feel free to ignore. Not sure if starting with the entity is the pattern but it reads bit repetitive. 

```suggestion
// Describes a Time-based One-time Password (TOTP) for a user.
```